docs/getting-started/online-quickstart.md

```diff
@@ -51,9 +51,9 @@ Edit imodeljs-samples/interactive-app/basic-viewport-app/.env.local
 
 ## Next Steps
 
-### [Follow tutorials to dive deeper into iModel.js]($docs/learning/tutorials/index.md)
+### [Follow tutorials to dive deeper into iTwin.js]($docs/learning/tutorials/index.md)
 
-### [iModel.js Blog](https://medium.com/imodeljs)
+### [iTwin.js Blog](https://medium.com/imodeljs)

```

Comment:
Here as well.

Reply:
@calebmshafer how about the Twitter feed in `CommunityResources.md`? Has that been changed?

Reply:
We don't currently have the itwinjs or itwin Twitter feeds, just imodeljs so we probably shouldn't switch that one yet.

Reply:
Just sent a message to a few people about locking down one, or both, of those handles.